lib/cell.rb

```diff
@@ -0,0 +1,44 @@
+class Cell < Ship ## concept of inheritance. The cell class inherits all functionality of the ship class.
```

Comment:
Awesome job figuring out the inheritance method. We both new that we were supposed to be able to use methods from the ship class. Sienna searched around and was able to figure that out.